packages/backend/server/src/models/base.ts

```diff
@@ -16,13 +15,12 @@ export class BaseModel {
   @Inject(Config)
   protected readonly config!: Config;
 
-  @Inject(PrismaClient)
-  protected readonly db!: PrismaClient;
-
   @Inject(TransactionHost)
   private readonly txHost!: TransactionHost<TransactionalAdapterPrisma>;
 
   protected get tx() {
```

Comment:
rename to `db`

Reply:
![Added via Giphy](https://media1.giphy.com/media/3ohzdQ3TjcSdJqQa52/giphy.gif?cid=c9487743vem0rurho2vyselhgjpjxy30v09beuook3ndwk6q&ep=v1_gifs_search&rid=giphy.gif&ct=g)